ietf/templates/doc/review_assignment_summary.html

```diff
@@ -7,7 +7,8 @@
                 {% if review_assignment.review_request.doc.name != current_doc_name %}
                     {{ review_assignment.review_request.doc.name }}
                 {% endif %}
-                -{{ review_assignment.reviewed_rev }}){% endif %}</a>
+                -{{ review_assignment.reviewed_rev }}){% endif %}
+            by {{ review_assignment.reviewer.person.name }}</a>
```

Comment:
Consider using plain_name here instead?

Reply:
Can do, but why?

Reply:
Because that's what we use fairly consistently everywhere else on pages like this. And it would be strange when someone's name appears twice on a page (for different reasons) to see a different form of the name.

I use this anecdote to help figure out which to use:
```
>>> ba = Person.objects.get(name__endswith=" Aboba")
>>> ba.plain_name()
'Bernard Aboba'
>>> ba.name
'Dr. Bernard D. Aboba'
```